packages/blockchain-wallet-v4-frontend/src/modals/CoinifyTradeDetails/Trade.js

```diff
@@ -84,6 +116,62 @@ const Trade = ({ trade, close, status }) => {
             <FormattedMessage id='orderdetails.footnote' defaultMessage='*Please note: depending on your bank’s tranfers policies, you will see the funds reflected in your account within 1-2 days from the transfer. ' />
           </Text>
         }
+        {
```

Comment:
All this recurring trade specific code should probably be in a separate file

Reply:
moved!